stdlib/statistics.pyi

```diff
@@ -79,6 +79,7 @@ def stdev(data: Iterable[_NumberT], xbar: _NumberT | None = None) -> _NumberT: .
 def variance(data: Iterable[_NumberT], xbar: _NumberT | None = None) -> _NumberT: ...
 
 class NormalDist:
+    __slots__ = {"_mu": "Arithmetic mean of a normal distribution", "_sigma": "Standard deviation of a normal distribution"}
```

Comment:
This is a little weird; the values are used as documentation for the slots. My tool just copied over the slots as is, but maybe we don't want the values in typeshed, but then how do we preserve the type of `__slots__`? I don't have a strong opinion.

Reply:
There's also a case like this in `tarfile` but I backed it out of this PR because the linter was complaining that the strings were too long. I can add it in a separate PR later.